pkg/logs/client/http/destination.go

```diff
@@ -86,9 +89,29 @@ func (d *Destination) Send(payload []byte) error {
 	}
 }
 
-// SendAsync is not implemented for HTTP.
+// SendAsync sends a payload in background.
 func (d *Destination) SendAsync(payload []byte) {
-	return
+	d.once.Do(func() {
+		payloadChan := make(chan []byte, 100)
+		d.sendInBackground(payloadChan)
+		d.payloadChan = payloadChan
+	})
+	d.payloadChan <- payload
+}
```

Comment:
Should we instrument the dropped messages here ? https://github.com/DataDog/datadog-agent/blob/master/pkg/logs/client/tcp/destination.go#L92

Reply:
Hmm never mind because of batching we won't know how many logs are in a payload 🤔